client/post-editor/media-modal/index.scss

```diff
@@ -127,6 +127,13 @@
 	}
 }
 
+.editor-media-modal-edit__content {
+	@extend .editor-media-modal__content;
```

Comment:
It's not a very good practice to reuse CSS that is not being inherited by means of reusing components.